### PR TITLE
RHDEVDOCS-3095: Cluster Logging Updates

### PR DESCRIPTION
**Pull request by libander**
- Aligned team: Dev Tools
- For branches: 4.7+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3095
- Direct link to doc preview: https://deploy-preview-36314--osdocs.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-log-store
- SME review: @alanconway 
- QE review: @kabirbhartiRH 
- Peer review: @rolfedh 
- Please merge

### CODE REVIEW

**Review by alanconway — COMMENTED** [review]
/lgtm

**Review by rolfedh — APPROVED** [review]

File: modules/cluster-logging-logstore-limits.adoc [hunk 1]
```diff
@@ -54,21 +56,25 @@ spec:
 ----
 <1> Specify the CPU and memory requests for Elasticsearch as needed. If you leave these values blank,
 the OpenShift Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are `16Gi` for the memory request and `1` for the CPU request.
-<2> Specify the CPU and memory requests for the Elasticsearch proxy as needed. If you leave these values blank, the OpenShift Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are `256Mi` for the memory request and `100m` for the CPU request.
+<2> The maximum amount of resources a Pod may use.
+<3> The minimum resources required to schedule a Pod.
```
Comment (abrennan89) [inline modules/cluster-logging-logstore-limits.adoc:60]: There should be "pod" I think, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references

**Review by abrennan89 — CHANGES_REQUESTED** [review]
@libander one small nit before merging, also can you please add a title for this PR

**Review by abrennan89 — APPROVED** [review]

**Pull request merged**